charts/epinio/100.0.0+up0.9.1/Chart.yaml

```diff
@@ -0,0 +1,44 @@
+annotations:
+  artifacthub.io/license: Apache-2.0
+  catalog.cattle.io/auto-install: epinio-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Epinio
+  catalog.cattle.io/experimental: "true"
+  catalog.cattle.io/kube-version: '>= 1.20 <= 1.23.6'
```

Comment:
Can you replace this for `'>= 1.20.0-0 < 1.24.0-0'`? That should cover all versions from 1.20 through 1.23 which seems like it is what this chart supports.

Reply:
Done with commit 4cd4e02fc3a4244b6e6b50a352c1b198eb025bbb (history compressed)